scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,588 @@
+# Ansible playbook to install OpenKAT
+#
+# Copyright (C) 2023  Jeroen Baten <jbaten@i2rs.nl>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301, USA
+#
+# Changelog
+# Date       Author   Comments
+# ===========================================================================
+# 2023-08-12 J. Baten Initial version
+# 2023-08-14 J. Baten Debugging. Hacking Superadmin password to support 
+#                     current stable standard Ansible
+# 2023-09-16 J. Baten Inject variables for version numers, and prepare for MR.                    
+
+# Usage; Make sure you can ssh to the root account on the remote server using
+#        authorized_keys. After that run the following command (and don't forget
+#        the comma!
+#
+#        ansible-playbook -vvv install-openkat.yml -i <hostname or ip>, -u root
+
+---
+
+- name: Playbook to install OpenKAT 1.10.1 on Debian 12
+  hosts: all
+
+  vars:
+    katrelease: "1.11.0"
+    xtdbrelease: "1.0.7"
+
+  tasks:
+
+  # Downloading and installing
+  # Download the packages for all the components of KAT from GitHub. Also download the XTDB multinode package from GitHub.
+
+  - name: "Download release {{ katrelease }} to our Debian 12 server"
+    get_url:
+      url: "https://github.com/minvws/nl-kat-coordination/releases/download/v{{ katrelease }}/kat-debian12-{{ katrelease }}.tar.gz"
+      dest: /tmp
+      mode: '0440'
+
+  - name: "Download release xtdb {{ xtdbrelease }} to our Debian 12 server"
+    get_url:
+      url: "https://github.com/dekkers/xtdb-http-multinode/releases/download/v{{ xtdbrelease }}/xtdb-http-multinode_{{ xtdbrelease }}_all.deb"
+      dest: /tmp
+      mode: '0440'
```

Comment:
URL's aanpassen indien de machine debian 11 / 12 / ubuntu draait ?

Reply:
Zou ik wel willen, maar zolang de url van de laatste productierelease niet voorspelbaar is (bijvoorbeeld 'latest') zal ik me moeten behelpen. En dit playbook is een eerste aanzet. Als je  debian 11 / 12 / ubuntu support wil dan zal dan versie 2.0 moeten worden. Eerst maar eens kijken hoe dit gaat.

Reply:
Je kan de versie van de releases/latest pagina krijgen. Dit is omzetting naar KAT van wat ik in een van m'n playbooks heb staan, geen idee of fact zetten eigenlijk de beste manier is, volgens mij heb ik het ooit ergens van gekopieerd, het werkt in ieder geval wel. Het openkat install script doet iets vergelijkbaars:

```
    - name: Get latest version
      shell: 'curl -sL https://github.com/minvws/nl-kat-coordination/releases/latest | grep -m 1 -Po "(?<=tag\/)v[0-9\.]*(?=\") | sed 's/^v//'"
      args:
        warn: false
      register: openkat_curl_output

    - name: Set version fact
      set_fact:
        openkat_version: "{{ openkat_curl_output.stdout }}"
```

En dan wordt de url

```
https://github.com/minvws/nl-kat-coordination/releases/download/v{{ openkat_version }}/kat-debian12-{{ openkat_version }}.tar.gz
```